src/stylesheets/loginView.scss

```diff
@@ -0,0 +1,43 @@
+.login {
+  &_layout {
+    text-align: left;
+    margin: 5% 10%;
+    width: 100%;
+  }
+
+  .welcome {
+    font-size: 4em;
+    padding-bottom: 1.5vw;
+  }
+
+  &_signup_link {
+    color: #135ef2;
```

Comment:
For other reviewers, there were issues using the $primary variable that we couldn't figure out how to resolve. This will be addressed before merging into main.